helm/fiftyone-teams-app/README.md

```diff
@@ -571,7 +571,13 @@ upgrading from FiftyOne Teams version 1.1.0 or later:
 A minimal example `values.yaml` may be found
 [here](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/values.yaml).
 
-1. Edit the `values.yaml` file
+1. Edit the `values.yaml` file, setting the required parameters including
+      1. `secret.fiftyone.mongodbConnectionString`
+      1. `secret.fiftyone.cookieSecret`
+      1. `secret.fiftyone.encryptionKey`
+      1. `secret.fiftyone.fiftyoneAuthSecret`
+      1. `apiSettings.dnsName`
```

Comment:
not actually required... they could use path-based routing.

What about all the auth0 settings if you're in legacy mode?

None of those secrets actually have to be set if `secret.create` is `false` - then they might just need `secret.name`

🤷 I mean, it's fine, but your next question is going to be "why do I have to set those if I made my own secret" - we don't set them in our `values.yaml` files.

Reply:
I was under the impression that you are already working on docs changes for v1.6.0. For that reason, I didn't include the auth modes introduced in v1.6.0.  I only used that branch because it might be out next release branch.

If we apply the one size fits most, this works. Advanced users who opt in to setting their own secret, will still need to provide those values, so they are still "required".  If you can come up with better wording, please suggest it here.

Reply:
How does it look now?